### PR TITLE
REG-390: deduplicate yarn.lock

### PR DESCRIPTION
**Pull request by ivanShapovalov**

### CODE REVIEW

**Review by aquiladev — COMMENTED** [review]
I have number of changes locally after `yarn install`
```
-"@babel/helper-validator-identifier@^7.12.11", "@babel/helper-validator-identifier@^7.14.5":
+"@babel/helper-validator-identifier@^7.14.5":
...
```

**Comment by ivanShapovalov** [comment]
> I have number of changes locally after `yarn install`
> 
> ```
> -"@babel/helper-validator-identifier@^7.12.11", "@babel/helper-validator-identifier@^7.14.5":
> +"@babel/helper-validator-identifier@^7.14.5":
> ...
> ```

@aquiladev, that's quite natural. For yarn to keep the versions exactly like they are in lock file one should be running `yarn install --frozen-lockfile` just like it is being done on ci.

**Comment by mvlabat** [comment]
The `yarn install` command updates `yarn.lock` if the current lock file is not enough to satisfy the dependencies. If running `yarn install` generates a diff, I'm afraid that the deduplicate tool isn't working correctly

**Comment by mvlabat** [comment]
```
-"@ethersproject/constants@^5.4.0":
+"@ethersproject/constants@>=5.0.0-beta.128", "@ethersproject/constants@^5.0.4", "@ethersproject/constants@^5.1.0", "@ethersproject/constants@^5.4.0":
```

This change is definitely incorrect: by no means beta versions can be considered compatible with stable ones. I'm getting more convinced that this tool doesn't work correctly, sadly.

**Comment by mvlabat** [comment]
```
-es-abstract@^1.18.2:
+es-abstract@^1.18.0-next.1, es-abstract@^1.18.0-next.2, es-abstract@^1.18.2:
```

This change is definitely incorrect: by no means pre-release versions can be considered compatible with stable ones. I'm getting more convinced that this tool doesn't work correctly, sadly.

**Comment by ivanShapovalov** [comment]
> ```
> -es-abstract@^1.18.2:
> +es-abstract@^1.18.0-next.1, es-abstract@^1.18.0-next.2, es-abstract@^1.18.2:
> ```
> 
> This change is definitely incorrect: by no means pre-release versions can be considered compatible with stable ones. I'm getting more convinced that this tool doesn't work correctly, sadly.
> 
> Upd. judging from their readme, they support pre-release versions, but apparently they don't recognize `next` as a valid word for it.

@mvlabat
I have taken closer look. We currently have two entries for es-abstract in our lock:
1. `es-abstract@^1.18.0-next.1, es-abstract@^1.18.0-next.2` resolved to "1.18.0"
and 
2. `es-abstract@^1.18.2` resolved to `version "1.18.3"`

As you can see neither in fact resolved to a pre-release version.
What deduplication did is made one entry resolving to `version "1.18.3"` for all three deps.

So, basically I would say this is for good.

**Comment by mvlabat** [comment]
Oh, you're right. Well, it's quite weird that `yarn` resolved them to a stable version O_o

**Review by aquiladev — APPROVED** [review]
LGTM, no conflicts after `yarn install`
